frontend/src/components/Tiers.js

```diff
@@ -107,23 +124,42 @@ export default class Tiers extends Component {
                 {button}
               </AsyncButton>
             }
-            {hasStripe && !hasPaypal && ( // paypal has priority -> to refactor after prototyping phase
-              <StripeCheckout
-                token={(token) => ::this.onTokenReceived(tier, {amount, interval, currency, token})}
-                stripeKey={stripeKey}
-                name={collective.name}
-                currency={currency}
-                bitcoin={collective.settings.bitcoin}
-                amount={convertToCents(amount)}
-                description={stripeDescription}>
-                  <AsyncButton
-                    color='green'
-                    inProgress={inProgress} >
-                    {button}
-                  </AsyncButton>
-              </StripeCheckout>
-            )}
 
+            {hasStripe && !hasPaypal && (
```

Comment:
don't think we need it either - didn't want to touch the old logic. It's more likely to cause bugs and we are going to retiring this code base anyway.